src/PlaywrightSharp.Tests/Launcher/LaunchWebSocketOptionTests.cs

```diff
@@ -48,7 +49,7 @@ public async Task ShouldSupportTheWebSocketOption()
         ///<playwright-file>launcher.spec.js</playwright-file>
         ///<playwright-describe>Playwright.launch |webSocket| option</playwright-describe>
         ///<playwright-it>should fire "disconnected" when closing with webSocket</playwright-it>
-        [Retry]
+        [SkipBrowserAndPlatformFact(skipFirefox: true, skipLinux: true)]

```

Comment:
this should have a comment mentioning it would be fixed in #622

Reply:
Not necessary. We can track that here in GitHub.